/usr/include/obs/obs-internal.h references caption/caption.h but make install does not install that header

obs-studio 26.1.0

as the subject line says the headers are missing in the make install target or should not be pulled by the obs-internal header.

We found this by trying to build the obs gphoto2 plugin for openSUSE.

Same problem here, make stops:
```
 In file included from /obs-studio/libobs/obs-scene.h:21,
/obs-studio/libobs/obs-internal.h:39:10: fatal error: caption/caption.h: Aucun fichier ou dossier de ce type
 #include <caption/caption.h>
          ^~~~~~~~~~~~~~~~~~~
```

--- a/deps/libcaption/CMakeLists.txt
+++ b/deps/libcaption/CMakeLists.txt
@@ -36,6 +36,7 @@ set(CAPTION_HEADERS
 )
 
 add_library(caption STATIC ${CAPTION_SOURCES})
+install_obs_headers("caption/caption.h")
 set_target_properties(caption PROPERTIES
 	FOLDER "deps"
 	POSITION_INDEPENDENT_CODE ON)

